modules/publication/README.md

```diff
@@ -0,0 +1,45 @@
+# Publication
+
+## Purpose
+The publication module allows users to propose research projects based on LORIS 
+data, upload publication media, and allows lead study PIs to review the project 
+proposals.
+
+## Intended Users
+
+Publication module is intended for site PIs to propose specific research projects
```

Comment:
just users, not "Site PIs" (and you can take out the word "lead" from the other PIs after making that change.)